Build fail - implicit declaration of function ‘vld1q_s8_x4’ on Armv7-linux with the latest release

I wanted to build binaries for ARMv6,7 on Linux but I'm getting the following two errors:

> [19:53:33] /workspace/srcdir/llama.cpp/ggml-quants.c: In function ‘ggml_vec_dot_iq2_xxs_q8_K’:
> --
>   | [19:53:33] /workspace/srcdir/llama.cpp/ggml-quants.c:7264:19: error: implicit declaration of function ‘vld1q_s8_x4’; did you mean ‘vld1q_s64’? [-Werror=implicit-function-declaration]
>   | [19:53:33]  7264 \|             q8b = vld1q_s8_x4(q8); q8 += 64;
>   | [19:53:33]       \|                   ^~~~~~~~~~~
>   | [19:53:33]       \|                   vld1q_s64
>   | [19:53:33] /workspace/srcdir/llama.cpp/ggml-quants.c:7264:19: error: incompatible types when assigning to type ‘int8x16x4_t’ from type ‘int’
>   | [19:53:33] cc1: some warnings being treated as errors
> 

They both seem to stem from the changes made in [SOTA 2-bit PR](https://github.com/ggerganov/llama.cpp/pull/4773).
I know that there was a reference that ARM NEON isn't supported yet, but I thought the above would still build.

I haven't found any way to build without the quants (eg, equivalent of the previous `LLAMA_K_QUANTS=OFF`).

Steps to reproduce:

```
cmake .. \
    -DCMAKE_INSTALL_PREFIX=$prefix \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=RELEASE \
    -DBUILD_SHARED_LIBS=ON \
    -DLLAMA_BUILD_TESTS=OFF \
    -DLLAMA_BUILD_EXAMPLES=ON \
    -DLLAMA_NATIVE=OFF \
    -DLLAMA_ACCELERATE=ON \
    -DLLAMA_AVX=ON \
    -DLLAMA_AVX2=ON \
    -DLLAMA_F16C=ON \
    -DLLAMA_FMA=ON \
    -DLLAMA_BLAS=OFF \
    -DLLAMA_CUBLAS=OFF \
    -DLLAMA_CLBLAST=OFF
```


**Version**: Latest master (b1794)
**System**: build system for armv7l-linux-gnueabihf-cxx03 (C compiler 10.2.0)
**Build log**: https://buildkite.com/julialang/yggdrasil/builds/7645#018ceaa2-2f2b-410c-8e4c-826c8b6a48a9
(all logs are here: https://buildkite.com/julialang/yggdrasil/builds/7645#_)

Thank you for your advice!

PS: The error is similar to [this one](https://github.com/ggerganov/llama.cpp/issues/3880), but I don't believe it's related (different arch)

--- a/ggml-quants.c
+++ b/ggml-quants.c
@@ -7261,7 +7261,7 @@ void ggml_vec_dot_iq2_xxs_q8_K(const int n, float * restrict s, const void * res
         const int8_t   * restrict q8 = y[i].qs;
         float sumf1 = 0, sumf2 = 0;
         for (int ib32 = 0; ib32 < QK_K/32; ib32 += 2) {
-            q8b = vld1q_s8_x4(q8); q8 += 64;
+            q8b = ggml_vld1q_s8_x4(q8); q8 += 64;
             memcpy(aux32, q2, 4*sizeof(uint32_t)); q2 += 8;
             q2u.val[0] = vcombine_s8(vld1_s8((const void *)(iq2xxs_grid + aux8[ 0])), vld1_s8((const void *)(iq2xxs_grid + aux8[ 1])));
             q2u.val[1] = vcombine_s8(vld1_s8((const void *)(iq2xxs_grid + aux8[ 2])), vld1_s8((const void *)(iq2xxs_grid + aux8[ 3])));

--- a/ggml-quants.c
+++ b/ggml-quants.c
@@ -7250,9 +7250,9 @@ void ggml_vec_dot_iq2_xxs_q8_K(const int n, float * restrict s, const void * res
     uint32_t aux32[4];
     const uint8_t * aux8 = (const uint8_t *)aux32;
 
-    int8x16x4_t q2u;
-    int8x16x4_t q2s;
-    int8x16x4_t q8b;
+    ggml_int8x16x4_t q2u;
+    ggml_int8x16x4_t q2s;
+    ggml_int8x16x4_t q8b;
 
     float sumf = 0;
     for (int i = 0; i < nb; ++i) {

